skiko/src/jvmMain/objectiveC/macos/drawlayer.m

```diff
@@ -81,64 +81,37 @@ - (void) makeFullscreen: (BOOL) value
 
 extern jboolean Skiko_GetAWT(JNIEnv* env, JAWT* awt);
 
-JNIEXPORT void JNICALL Java_org_jetbrains_skiko_HardwareLayer_init(JNIEnv *env, jobject canvas)
+JNIEXPORT void JNICALL Java_org_jetbrains_skiko_HardwareLayer_nativeInit(JNIEnv *env, jobject canvas, jlong platformInfoPtr)
 {
     if (layerStorage == nil)
     {
         layerStorage = [[NSMutableSet alloc] init];
```

Comment:
Couldn't we get rid of `layerStorage` global and pass it explicitly from Kotlin side?

Reply:
We can, but it should be done in another PR, these changes don't fit this PR (it is only about getAWT/getDrawingSurface) and I am afraid to break this :)

Reply:
OK.